assets/pages/regular-contributions/components/regularContributionsPayment.jsx

```diff
@@ -57,11 +57,9 @@ function getStatusMessage(
     return <ProgressMessage message={['Processing transaction', 'Please wait']} />;
   } else if (hide) {
     return <ErrorMessage message="Please fill in all the fields above." />;
-  } else if (error !== null && error !== undefined) {
-    return <ErrorMessage message={error} />;
   }
 
-  return null;
+  return <ErrorMessage message={error} />;
```

Comment:
The logic of this function is being changed here before it didn't return always an errorMessage, and now it does. Is it intentional?

Reply:
Yup, now the "if error then show `ErrorMessage` else return `null`" logic is handled by `ErrorMessage`.